CONTRIBUTING.md

```diff
@@ -72,6 +72,10 @@ Packetbeat:
 Some of the Beats might have extra development requirements, in which case you'll find a
 CONTRIBUTING.md file in the Beat directory.
 
+We use an [EditorConfig](http://editorconfig.org/) file in the beats repository to standardise 
+how different editors use whitespaces etc. in our files. Most popular editors have a
```

Comment:
I guess tabs are technically whitespace...hehe. but I think you see the point...